### ISSUE TITLE
I can't run train.py since rnn_cell cannot be imported

### ISSUE THREAD

**Issue by veer66** [zakizhou/RNN-Tagger#2]
After running train.py, I got the error message as follow: 

> from tensorflow.python.ops import rnn_cell, rnn
> ImportError: cannot import name rnn_cell

It seems like  tensorflow.python.ops.rnn_cell is absent from tensorflow r1.0

**Comment by zakizhou** [comment]
@veer66 yeah you're right, there will be many changes to apis related with rnn and seq2seq in comming r1.1, so all rnn_cells were moved to contrib module, but in r.012 my it should work fine.
